### ISSUE TITLE
Screenshot links should have dates for screen readers

### ISSUE THREAD

**Issue by schwartzadev** [COVID19Tracking/website#731]
Let us know what the design issue is. Screenshots are helpful!

The screenshot links on state pages (i.e. [Arizona](https://covidtracking.com/data/state/arizona/)) currently don't have any labeling for screen readers to indicate that they belong to a particular date.

This ambiguity is compounded by multiple links with the same name in one table (i.e. `4:05 am`).

I _think_ this is an issue, but I'm tapping The Expert @beep for confirmation.

**Comment by beep** [comment]
@schwartzadev Hm! I think…this is _probably_ fine, given that a.) this is tabular data and b.) there aren’t many _sequential_ links with the `same name, different destination` problem.

But that’s just a gut read: I’d say that this is something we could get some feedback on from users.

Two quick, slightly-tangential suggestions that might be useful here:

- The first cell in each row could be a `th` with a `scope="row"` attribute.
- The `Screenshot` header should be labeled `Screenshots`, since most rows have multiple entries.

**Comment by kevee** [comment]
Agree with @beep that it should be fine since it is tabular and the user probably moved over to the screenshots from the first cell. I like the idea of creating a `th` for the date.

**Comment by schwartzadev** [comment]
Works for me!
